src/Modules/OrchardCore.Commerce/Constants/OrchardCoreCommerceConfigureOrder.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.Commerce.Constants;
+
+public static class OrchardCoreCommerceConfigureOrder
+{
+    public const int Default = 0;
```

Comment:
Orchard core defines the default. I think you should use that default and build on the top of it. You should not define your own default just in case OC ever changes the default.

https://github.com/OrchardCMS/OrchardCore/blob/84ae5cb0c8e0ec297a645f32b39695c3ea7003a3/src/OrchardCore/OrchardCore.Abstractions/OrchardCoreConstants.cs#L34

Reply:
I kept the constant, but [set its value to `OrchardCoreConstants.ConfigureOrder.Default`](https://github.com/OrchardCMS/OrchardCore.Commerce/pull/454/commits/2babc78bd54204af4b9fd0563147ac2f725cc8d0#diff-663021e292817e133e73c1e0f90e566008432df078580a21d9f4e754ae5c4d75R5) so any other constants are defined in relation to the one from OC.